Emby.Server.Implementations/EntryPoints/UserDataChangeNotifier.cs

```diff
@@ -145,13 +147,13 @@ private UserDataChangeInfo GetUserDataChangeInfo(Guid userId, List<BaseItem> cha
 
         public void Dispose()
         {
-            if (UpdateTimer != null)
+            if (_updateTimer != null)
             {
-                UpdateTimer.Dispose();
-                UpdateTimer = null;
+                _updateTimer.Dispose();
+                _updateTimer = null;
             }
```

Comment:
```c#
_updateTimer?.Dispose();
_updateTimer = null;
```
?

Reply:
Wht? We check the line before if it's not null.

Reply:
because it's shorter? Idc much tho

Reply:
Oh, yeah we could...
Doesn't really matter tho, does it?